packages/react-core/src/components/FileUpload/FileUploadField.tsx

```diff
@@ -138,16 +143,15 @@ export const FileUploadField: React.FunctionComponent<FileUploadFieldProps> = ({
               name={`${id}-filename`}
               aria-label={filenameAriaLabel}
               placeholder={filenamePlaceholder}
-              aria-describedby={`${id}-browse-button`}
               value={filename}
             />
           </InputGroupItem>
           <InputGroupItem>
             <Button
-              id={`${id}-browse-button`}
```

Comment:
We could keep this in, but if we're going to expose an ID on the browse button then we should consider allowing an ID to be passed to all other input elements within the component. For now I included this in the codemod issue https://github.com/patternfly/pf-codemods/issues/551